### PR TITLE
Bring Maven Wrapper current

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
@Courtneyp123 @rvema @friederbluemle please review

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ install: true
 before_script:
 - sudo chown -R $USER:$GROUP $TRAVIS_BUILD_DIR
 script:
-- if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$TRAVIS_BRANCH" = "master" ]; then mvn clean install -q -U; fi
+- if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$TRAVIS_BRANCH" = "master" ]; then ./mvnw clean install -q -U; fi
```
Comment (rvema) [inline .travis.yml:17]: please revert to `mvn` as we dont want to use mvn wrapper for travis builds.

**Review by rvema — CHANGES_REQUESTED** [review]
please address the review comment

**Review by friederbluemle — CHANGES_REQUESTED** [review]
Please keep CRLF endings in `mvnw.cmd` - A `.cmd` file is used on Windows _only_.

File: .travis.yml (hunk 1 above)
Comment (gliptak) [reply .travis.yml]: I updated as requested.

**Comment by gliptak** [comment]
@friederbluemle I think I added it with CRLF ...

**Review by friederbluemle — APPROVED** [review]
OK looks good now. Thanks @gliptak :+1:

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**